docs: Update platform helm chart to latest (#1660)
seqeralabs/docs: platform-enterprise_docs/enterprise/install-groundswell.md

| Docker Compose | [Pipeline optimization: Docker Compose](./groundswell-docker-compose) |

See each deployment guide for detailed requirements.

> An example to configure TLS connection to the pipeline optimization databases is available in the [helm charts repository](https://github.com/seqeralabs/helm-charts/tree/master/charts/platform/examples).

Reviewer: ```suggestion
For an example TLS configuration for the pipeline optimization databases, see the [helm charts repository](https://github.com/seqeralabs/helm-charts/tree/master/charts/platform/examples).
```
